easybuild/tools/systemtools.py

```diff
@@ -152,6 +160,56 @@
 RPM = 'rpm'
 DPKG = 'dpkg'
 
+SYSTEM_TOOLS = {
+    '7z': "extracting sources (.iso)",
+    'bunzip2': "decompressing sources (.bz2, .tbz, .tbz2, ...)",
+    DPKG: "checking OS dependencies (Debian, Ubuntu, ...)",
+    'gunzip': "decompressing source files (.gz, .tgz, ...)",
+    'make': "build tool",
+    'patch': "applying patch files",
+    RPM: "checking OS dependencies (CentOS, RHEL, OpenSuSE, SLES, ...)",
+    'sed': "runtime patching",
+    'Slurm': "backend for --job (sbatch command)",
+    'tar': "unpacking source files (.tar)",
+    'unxz': "decompressing source files (.xz, .txz)",
+    'unzip': "decompressing files (.zip)",
+}
+
+SYSTEM_TOOL_CMDS = {
+    'Slurm': 'sbatch',
+}
+
+OPT_DEPS = {
+    'archspec': "determining name of CPU microarchitecture",
+    'autopep8': "auto-formatting for dumped easyconfigs",
+    'GC3Pie': "backend for --job",
```

Comment:
We used this through EB installation when we used it... and we do have it in EB...

Reply:
Doesn't matter much, the point is to check all for optional dependencies whether they're available or not, it doesn't matter how they're installed...